lib/brcobranca/boleto/template/rghost.rb

```diff
@@ -139,8 +139,10 @@ def modelo_generico_cabecalho(doc, boleto)
           doc.show "#{boleto.banco}-#{boleto.banco_dv}", tag: :grande
           doc.moveto x: '7.5 cm', y: '23.85 cm'
           doc.show boleto.codigo_barras.linha_digitavel, tag: :grande
-          doc.moveto x: '0.7 cm', y: '23 cm'
+          doc.moveto x: '1.9 cm', y: '23.4 cm'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.